proto/schema/v1/tx.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+package sonrio.sonr.schema;
+
+import "schema/v1/schema.proto";
+
+// this line is used by starport scaffolding # proto/tx/import
+
+option go_package = "github.com/sonr-io/sonr/x/schema/types";
+
+// Msg defines the Msg service.
+service Msg {
+  rpc CreateSchema(MsgCreateSchema) returns (MsgCreateSchemaResponse);
+  rpc DeprecateSchema(MsgDeprecateSchema) returns (MsgDeprecateSchemaResponse);
+}
+
+message MsgCreateSchema {
+  string creator = 1;
+  string label = 2;
+  map<string, SchemaKind> fields = 3;
+}
+
+message MsgCreateSchemaResponse {
+  Schema schema = 1;
+}
+
+message MsgDeprecateSchema {
+  string creator = 1;
+  string did = 2;
+}
+
+message MsgDeprecateSchemaResponse {}
```

Comment:
Should we provide a status in this response? an empty message type feels a bit weird to me.

Reply:
Yes good point. I updated both of the response models to match the other modules (added code and message fields).